tests/atoms/gaia_data_layer.js

```diff
@@ -66,16 +67,17 @@ var GaiaDataLayer = {
 
   enableBluetooth: function() {
     var bluetooth = window.navigator.mozBluetooth;
-    if (!bluetooth.enabled) {
+    if (bluetooth.defaultAdapter.state != 'enabled') {
```

Comment:
`bluetooth.defaultAdapter.state === 'disabled' `+  same comment as above for 'disabling'